renaming_status.txt

```diff
@@ -0,0 +1,81 @@
+PROPOSED GITHUB REPO RENAME:    py-ipfs-api -> py-ipfs-http-client
+PROPOSED PIP MODULE RENAME:     ipfshttpclient
+PROPOSED PROJECT RENAME:        Python IPFS API -> Python IPFS HTTP Client
+External Changes to be made:
+(See the section under "FOR README.md")
+
+
+THESE MAY REFER TO THE ACTUAL API
+ipfshttpclient/client.py          All occurrences
+ipfshttpclient/encoding.py        Ln 392
+ipfshttpclient/http.py            Ln 2,4,134
+ipfshttpclient/version.py         Ln 7
+test/run-tests.py                 Ln 68,102
```

Comment:
Reviewed all these and all, except for the ones in `client.py`, indeed refer to the IPFS HTTP API and should be changed. Current state looks good.